holoviews/plotting/plot.py

```diff
@@ -719,6 +724,11 @@ def _compute_group_range(cls, group, elements, ranges, framewise, top_level):
                     data_range = el.range(el_dim, dimension_range=False)
 
                 data_ranges[(el, el_dim)] = data_range
+                if dtype is not None and dtype.kind == 'uif' and robust:
```

Comment:
Could you add a comment saying this comes from xarray and maybe point to the exact code this is inspired by?